src/app/pools/views/PoolList/PoolList.tsx

```diff
@@ -82,40 +80,29 @@ const generateRows = (rows: ResourcePool[]) =>
 
 const Pools = (): JSX.Element => {
   useWindowTitle("Pools");
-  const dispatch = useDispatch();
 
-  const poolsLoaded = useSelector(resourcePoolSelectors.loaded);
-  const poolsLoading = useSelector(resourcePoolSelectors.loading);
-  const errors = useSelector(resourcePoolSelectors.errors);
-  const errorMessage = formatErrors(errors);
-
-  useFetchActions([resourcePoolActions.fetch]);
-
-  const resourcePools = useSelector(resourcePoolSelectors.all);
+  const listPools = useListPools();
+  const errorMessage = formatErrors(listPools.error);
+  const listPoolsResources = listPools.data?.items || [];
```

Comment:
```suggestion
  const resourcePools = listPools.data?.items || [];
```

I think this is a clearer name, optional though